src/msg/async/dpdk/dpdk_rte.cc

```diff
@@ -137,8 +137,12 @@ namespace dpdk {
       if (!rte_initialized) {
         /* initialise the EAL for all */
         int ret = rte_eal_init(cargs.size(), cargs.data());
-        if (ret < 0)
-          return;
+	if (ret < 0) {
+          std::unique_lock<std::mutex> l(lock);
```

Comment:
```suggestion
          std::unique_lock locker(lock);
```

please avoid using a single `l` for the variable name. `l` looks like `1` with a bad font.

Reply:
Ok